packages/protocol/test/governance/vesting.ts

```diff
@@ -552,298 +548,333 @@ contract('Vesting', (accounts: string[]) => {
-      await vestingInstance.createAccount({ from: beneficiary })
+      await createNewReleaseGoldInstanceTx(releaseGoldDefaultSchedule, web3)
+      releaseGoldInstanceAddress = await releaseGoldFactoryInstance.releases(beneficiary, 0)
+      releaseGoldInstance = await ReleaseGoldInstance.at(releaseGoldInstanceAddress)
+      await releaseGoldInstance.createAccount({ from: beneficiary })
     })
 
     it('beneficiary should set dataEncryptionKey', async () => {
-      await vestingInstance.setAccountDataEncryptionKey(dataEncryptionKey, { from: beneficiary })
+      await releaseGoldInstance.setAccountDataEncryptionKey(dataEncryptionKey, {
+        from: beneficiary,
+      })
       // @ts-ignore
       const fetchedKey: string = await accountsInstance.getDataEncryptionKey(
-        vestingInstance.address
+        releaseGoldInstance.address
       )
       assert.equal(fetchedKey, dataEncryptionKey)
     })
 
     it('should revert if non-beneficiary attempts to set dataEncryptionKey', async () => {
       await assertRevert(
-        vestingInstance.setAccountDataEncryptionKey(dataEncryptionKey, { from: accounts[2] })
+        releaseGoldInstance.setAccountDataEncryptionKey(dataEncryptionKey, { from: accounts[2] })
       )
     })
 
     it('should allow setting a key with leading zeros', async () => {
       const keyWithZeros = '0x00000000000000000000000000000000000000000000000f2f48ee19680706191111'
-      await vestingInstance.setAccountDataEncryptionKey(keyWithZeros, { from: beneficiary })
+      await releaseGoldInstance.setAccountDataEncryptionKey(keyWithZeros, { from: beneficiary })
       // @ts-ignore
       const fetchedKey: string = await accountsInstance.getDataEncryptionKey(
-        vestingInstance.address
+        releaseGoldInstance.address
       )
       assert.equal(fetchedKey, keyWithZeros)
     })
 
     it('should revert when the key is invalid', async () => {
       await assertRevert(
-        vestingInstance.setAccountDataEncryptionKey('0x32132931293', { from: beneficiary })
+        releaseGoldInstance.setAccountDataEncryptionKey('0x32132931293', { from: beneficiary })
       )
     })
 
     it('should allow a key that is longer than 33 bytes', async () => {
-      await vestingInstance.setAccountDataEncryptionKey(longDataEncryptionKey, {
+      await releaseGoldInstance.setAccountDataEncryptionKey(longDataEncryptionKey, {
         from: beneficiary,
       })
       // @ts-ignore
       const fetchedKey: string = await accountsInstance.getDataEncryptionKey(
-        vestingInstance.address
+        releaseGoldInstance.address
       )
       assert.equal(fetchedKey, longDataEncryptionKey)
     })
   })
 
-  Object.keys(authorizationTestDescriptions).forEach((key) => {
-    describe('authorization tests:', () => {
-      let authorizationTest: any
-      let vestingInstanceAddress: any
-      let vestingInstance: any
+  describe('#setMaxDistribution', () => {
```

Comment:
Can you include a test for the 100% special case?

Reply:
Done, also added testing of rewards